docs/introduction/what_is_renku.rst

```diff
@@ -5,60 +5,109 @@ What is Renku?
 
 The Renku Project is a web platform (:ref:`renkulab`) and a command-line
 interface (:ref:`renku`) built on top of open source components for researchers,
-data scientists, educators, and students to help manage:
-
-* data
-* code
-* workflows
-* execution environments
+data scientists, educators, and students to help manage their **data**,
+**code**, **workflows**, and **computational environments**.
```

Comment:
well here "workflows" and "provenance" are sort of synonymous to me, but perhaps you're right to mention it explicitly since they do sort of have different implications.